rubicon_ml/sklearn/pipeline.py

```diff
@@ -135,3 +137,80 @@ def get_estimator_logger(self, step_name=None, estimator=None):
             logger.set_estimator(estimator)
 
         return logger
+
+
+def make_pipeline(
+    project,
+    *steps,
+    experiment_kwargs={"name": "RubiconPipeline experiment"},
+    memory=None,
+    verbose=False
+):
+    """Wrapper around RubicionPipeline(). Does not require naming for estimators
```

Comment:
docstring needs a little clean up - mainly removing `user_defined_loggers` and `kwargs` and replacing them with `memory` and `verbose`. you can just copy the docstrings from sklearn over to this (except the `:class:Pipeline` part - i dont think our docs build understands that syntax)
> https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/pipeline.py#L807